src/coprocessor/dag/expr/builtin_math.rs

```diff
@@ -450,6 +450,16 @@ impl ScalarFunc {
         let x = try_opt!(self.children[0].eval_real(ctx, row));
         Ok(Some(x * f64::consts::PI / 180_f64))
     }
+
+    #[inline]
+    pub fn exp(&self, ctx: &mut EvalContext, row: &[Datum]) -> Result<Option<f64>> {
+        let x = try_opt!(self.children[0].eval_real(ctx, row));
+        let r = x.exp();
+        if r.is_infinite() || r.is_nan() {
+            return Ok(Some(0_f64));
```

Comment:
Should we document this behavior or is it documented somewhere already? (invalid float points are converted into `0_f64`)

Reply:
actually, seems it should return an overflow error instead of 0, I will fix it tomorrow.
By the way , when user input "select exp("abc");" , what will tikv receive from tidb of the argument?  `Datum::F64(0)`, `Datum::Null`, or some other thing